doc/usage/configuration.rst

```diff
@@ -1333,6 +1333,15 @@ that use Sphinx's HTMLWriter class.
 
    .. versionadded:: 1.6
 
+Options for Single HTML output
+-------------------------------
+
+.. confval:: singlehtml_sidebars
+
+   Custom sidebar templates, must be a dictionary that maps document names to
```

Comment:
This should mention that the only valid key is `index` since that's the only page being built.

Reply:
done